### PR TITLE
Fix ordering to permit postMessage immediately after adoptPredecessor.

### PR DESCRIPTION
**Pull request by jeremyroman**
@a4sriniv PTAL

***
<a href="https://pr-preview.s3.amazonaws.com/jeremyroman/portals/pull/126.html" title="Last updated on May 16, 2019, 8:44 PM UTC (d3cd8de)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/portals/126/951367a...jeremyroman:d3cd8de.html" title="Last updated on May 16, 2019, 8:44 PM UTC (d3cd8de)">Diff</a>

### CODE REVIEW

**Comment by jeremyroman** [comment]
Resolves #97.

File: index.bs [hunk 1]
```diff
@@ -652,6 +655,13 @@ spec:url; type:dfn; for:/; text:url
 
     1. Run the steps to [=adopt the predecessor browsing context=] |predecessorBrowsingContext| in |successorWindow|, and return the result.
 
+    1. [=Queue a task=] to [=complete portal activation=].
+
+        Note: Queuing this immediately makes it possible to send messages to the
```
Comment (a4sriniv) [inline index.bs]: Should we say "adopted portal" instead of just "portal" to be clearer?

Comment (jeremyroman) [reply index.bs]: Done.

**Comment by jeremyroman** [comment]
PTAL

**Pull request merged**